.github/workflows/periodic.yml

```diff
@@ -0,0 +1,29 @@
+name: Periodic Functional Tests
+
+on:
+  schedule:
+    - cron: '*/5 * * * *' # Run every 5 minutes
+
+jobs:
+  run_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      - name: Create k8s Kind Cluster
+        uses: container-tools/kind-action@v2
+        with:
+          cluster_name: kfp-tekton
+          kubectl_version: v1.29.2
+          version: v0.22.0
+          node_image: kindest/node:v1.29.2
+      - name: Run Functional Tests
+        run: |
+          log_dir=$(mktemp -d)
+          ./test/kfp-functional-test/kfp-functional-test.sh > $log_dir/periodic_tests.txt
+      - name: Collect test results
+        if: always()
+        uses: actions/upload-artifact@v4
+        with:
+          name: periodic-functional-artifacts
+          path: $log_dir
```

Comment:
Replace this line with the same content as in https://github.com/kubeflow/pipelines/blob/master/.github/workflows/backend.yml#L81